site/components/form/TextInput.test.tsx

```diff
@@ -51,4 +51,19 @@ describe("TextInput", () => {
             .toJSON();
         expect(tree).toMatchSnapshot();
     });
+    it("should render correctly if input is disabled", () => {
+        const tree = renderer
+            .create(
+                <TextInput<TestInputs>
+                    inputName="field1"
+                    display="Test Field"
+                    stateUpdater={vi.fn()}
+                    widthClass="w-3/4"
+                    maxLength={50}
+                    disabled={true}
```

Comment:
It is signing me out when i change my name - i assumed it would only make sense to logout if you change the role?

Reply:
missed a bit of logic out on the edit-user api - fixed